tests/src/test/scala/org/apache/openwhisk/core/containerpool/test/ContainerPoolTests.scala

```diff
@@ -829,7 +842,7 @@ class ContainerPoolTests
     stream.reset()
     val prewarmExpirationCheckIntervel = 2.seconds
     val poolConfig =
-      ContainerPoolConfig(MemoryLimit.STD_MEMORY * 8, 0.5, false, prewarmExpirationCheckIntervel, None, 100)
+      ContainerPoolConfig(MemoryLimit.STD_MEMORY * 12, 0.5, false, prewarmExpirationCheckIntervel, None, 100)
```

Comment:
Due to calculate prewarmStartingPool's memory, so need to adjust the user memory here.